nodebalancer/cmd/nodebalancer/configs.go

```diff
@@ -159,6 +193,27 @@ func LoadConfig(configPath string) error {
 	return nil
 }
 
+func LoadContractsConfig(configPath string) error {
+
+	if configPath == "" {
```

Comment:
Should not we fail if config of contracts not provided?
What default config?
Can we have sample of default config?

Reply:
I was think make it more less restrict so if you want just nodebalancer functionality you can use it without contracts config.